pom.xml

```diff
@@ -167,5 +167,11 @@
       <version>2.12.4</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
```

Comment:
is it a unique dependency that does not exist within symphony core? 
does jackson not have the same functionality you need?

Reply:
Thanks, Mak. 
I found an option for Jackson itself 

`ObjectMapper mapper = new ObjectMapper();
mapper.enable(JsonParser.Feature.STRICT_DUPLICATE_DETECTION);`

I will fix it!